backend/test/integration/StateChangeEvents.integration.test.js

```diff
@@ -132,7 +175,44 @@ describe('DayStateEvents integration tests', () => {
         for (let i = 1; i < 6; i += 1) {
             await connectAndJoin(clientSockets, i, port, lobbyCode);
         }
-        await startGame(clientSockets);
+        await startGameSixPlayer(clientSockets);
+        await startDay();
+
+        done();
+    });
+
+    test('integration test start-day 7 players', async (done) => {
```

Comment:
Is there a way to reduce the duplicate code here and somehow pass in the number of players?

Reply:
Done and Done.